charts/rancher-monitoring/v0.1.1/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.4.0-rc1
```

Comment:
If we're replacing this as of 2.4.0-rc1, we should make monitoring v0.1.0 with a max version to remove it.